src/UglyToad.PdfPig.Core/StreamInputBytes.cs

```diff
@@ -14,6 +15,11 @@ public class StreamInputBytes : IInputBytes
 
         private bool isAtEnd;
 
+        /// <summary>
+        /// Bytes
+        /// </summary>
+        public IReadOnlyList<byte> Bytes => throw new NotImplementedException();
```

Comment:
I guess this might have to be a lazily evaluated property that reads the entire stream into memory for supported stream types `File`, `Memory` and throws otherwise (`HttpContent`)?